src/components/Completed/CompletedButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { setResourceIsCompleted } from '../../store/resourcesSlice';
+import RadioButtonUncheckedIcon from '@material-ui/icons/RadioButtonUnchecked';
```

Comment:
Just a note for the future; we will probably have icons from @dfangshuo's designs that we can directly input, but these are good for now!